src/cc/BPFTable.h

```diff
@@ -174,17 +179,18 @@ class BPFHashTable : public BPFTableBase<KeyType, ValueType> {
 
   std::vector<std::pair<KeyType, ValueType>> get_table_offline() {
     std::vector<std::pair<KeyType, ValueType>> res;
-
-    KeyType cur, nxt;
+    KeyType cur;
     ValueType value;
 
+    if (!this->first(&cur))
+      return res;
+
     while (true) {
-      if (!this->next(&cur, &nxt))
+      if (!this->lookup(&cur, &value))
```

Comment:
@drzaeus77 I would love too. But because the base class is templated and how symbol lookup works, I had to do either `this->lookup()` or `BPFHashTable<KeyType, ValueType>::lookup()`, or `BPFTableBase<KeyType, ValueType>::lookup()`. Do we prefer the later?